frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -71,8 +80,37 @@ common_info.report_bug = {
         else
             msg = common_msg
         end
-        UIManager:show(InfoMessage:new{
+        UIManager:show(ConfirmBox:new{
             text = msg,
+            icon = "notice-info",
+            no_ok_button = true,
+            other_buttons_first = true,
+            other_buttons = {{
+                {
+                    text = G_reader_settings:isTrue("debug_verbose") and _("Disable verbose logging") or _("Enable verbose logging"),
+                    callback = function()
+                        -- Flip verbose logging on dismissal
+                        -- Unlike in the dev options, we flip everything at once.
+                        if G_reader_settings:isTrue("debug_verbose") then
+                            dbg:setVerbose(false)
+                            dbg:turnOff()
+                            G_reader_settings:makeFalse("debug_verbose")
+                            G_reader_settings:makeFalse("debug")
+                            Notification:notify(_("Verbose logging disabled"), Notification.SOURCE_ALWAYS_SHOW)
+                        else
+                            dbg:turnOn()
+                            dbg:setVerbose(true)
+                            G_reader_settings:makeTrue("debug")
+                            G_reader_settings:makeTrue("debug_verbose")
+                            Notification:notify(_("Verbose logging enabled"), Notification.SOURCE_ALWAYS_SHOW)
```

Comment:
(Not really caring, but do you need these notifications now that you have it written in the menu item just under this?)